### PR TITLE
[SES-246] Connect cost breakdown with API

### PR DESCRIPTION
**Pull request by YasielCabrera**
# Ticket
https://trello.com/c/tqmW07KT/246-feature-aggregatedviewlevels-v1

# Description
Connect the UI of the Cost Breakdown table on the Finance's page with the API

# What solve
- Should get the required data from the API on Server Side
- Should have the Budget column: The core unit column will list core units and recognized delegates by name.
- Should add column “% of Total”
- Should be displayed 12 row, when the “By Budget” category is select
- Should order the entries of the table by absolute amount regardless of their source
- Should on the "By budget" the top 10 expenses be by absolute amount regardless of their source, either by individual core units, Individual recognized delegates , “remaining core units” and  “remaining recognized delegates”
- The "View" CTA should go to the right place
- Should have "Total Spend” with the absolute number that the line item is, expressed in DAI

### CODE REVIEW

File: src/stories/containers/FinancesOverview/useFinancesOverview.ts [hunk 1]
```diff
@@ -135,6 +140,53 @@ const useFinancesOverview = (quarterExpenses: ExpenseDto[] = [], monthly: Partia
   // cost breakdown
   const [selectedFilter, setSelectedFilter] = useState<CostBreakdownFilterValue>('By budget');
 
+  const { byBudgetExpenses, costBreakdownTotal, remainingBudgetCU, remainingBudgetDelegates } = useMemo(() => {
+    let costBreakdownTotal = 0;
+    const byBudgetExpenses: ExtendedExpense[] = [];
+    // const byCategory: ExtendedExpense[] = [];
```
[the remark this answers is not in the document]
Comment (YasielCabrera) [reply src/stories/containers/FinancesOverview/useFinancesOverview.ts:146]: No, I will remove it

File: pages/index.tsx [hunk 2]
```diff
@@ -1,38 +1,55 @@
...
 export default FinanceOverviewPage;
 
 export async function getServerSideProps() {
   if (featureFlags[CURRENT_ENVIRONMENT].FEATURE_FINANCES_OVERVIEW) {
-    const quarterExpenses = await fetchExpenses(ExpenseGranularity.quarterly);
-    const monthlyExpenses = await fetchExpenses(ExpenseGranularity.monthly);
+    const [quarterExpenses, monthlyExpenses, breakdownExpenses] = await Promise.all([
+      fetchExpenses(ExpenseGranularity.quarterly),
+      fetchExpenses(ExpenseGranularity.monthly),
+      // fetchExpenses(ExpenseGranularity.annual, 'makerdao/*:*/*:*'),
```
[the remark this answers is not in the document]
Comment (YasielCabrera) [reply pages/index.tsx:44]: No, I will remove it

**Comment by alejandrocabriales** [comment]
LGTM

**Comment by alejandrocabriales** [comment]
LGTM

**Review by alejandrocabriales — APPROVED** [review]
LGTM

**Pull request merged**